src/plugins/data/public/antlr/opensearch_ppl/code_completion.ts

```diff
@@ -40,16 +40,50 @@ export const getSuggestions = async ({
       finalSuggestions.push(...fetchFieldSuggestions(indexPattern, (f: any) => `${f} `));
     }
 
+    if (suggestions.suggestValuesForColumn) {
+      // get dataset for connecting to the cluster currently engaged
+      const dataset = services.data.query.queryString.getQuery().dataset;
+
+      // take the column and push in values for that column
+      const res = await fetchColumnValues(
+        [indexPattern.title],
+        suggestions.suggestValuesForColumn,
+        services,
+        dataset
+      );
+
+      let i = 0;
+      finalSuggestions.push(
+        ...res.body.fields[0].values.map((val: any) => {
+          i++;
+          return {
+            text: val.toString(),
+            insertText: typeof val === 'string' ? `"${val}" ` : `${val} `,
```

Comment:
do we need to consider use case like when user doesn't have permission to see values for certain field? like if the values for certain sensitive fields are masked.

Reply:
testing field masking